`npm run serve` fails

I cloned this repository and followed the steps in the `README`, specifically
- `nix develop`
- `npm install`
- `npm run build`

After those, `npm run serve` fails with
```
✘ [ERROR] Cannot serve an entry point without an output path
```
That seems to be hitting this path in `esbuild` https://github.com/evanw/esbuild/blob/a652e730ff07b9081470ef6965f3d54daa7b2aab/pkg/api/serve_other.go#L726

If I add `--outfile=dist/index.js` to the `"serve"` script in `package.json` that seems to fix things.
```diff
-  "serve": "spago build && esbuild --bundle index.js --servedir=dist",
+  "serve": "spago build && esbuild --bundle index.js --outfile=dist/index.js --servedir=dist",
```
I can make a PR with that change if it makes sense.

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "clean": "rm -rf node_modules output .spago dist/*.js *.lock .cache",
     "build": "spago build",
     "test": "spago test",
-    "serve": "spago build && esbuild --bundle index.js --servedir=dist",
+    "serve": "spago build && esbuild --bundle index.js --outfile=dist/index.js --servedir=dist",
     "bundle": "spago build --output output-es && purs-backend-es bundle-app --main Main --minify --no-build --to dist/index.js"
   },
   "dependencies": {

